src/databases/PVLD/avtPVLDFileFormat.C

```diff
@@ -374,21 +374,21 @@ avtPVLDFileFormat::GetMesh(int domain, const char *meshname)
                     preader_->ReadTiedSetBlockMesh( domain, crd, ele );
                     if( crd.size()>0 ) return GenerateShellMesh( crd, ele );
                 }
-                else if( vs[1] == "Master" )
+                else if( vs[1] == "Manager" )
```

Comment:
Ok, so this change and associated changes involving the hard coded strings `"Master"` and `"Slave"` will of course cause problems for proper operation of the reader because it is expecting those names in the associated HDF5 files. I

Reply:
I have corrected this issue.